pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -578,6 +587,13 @@ func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg
 	m.recorder.AnnotatedEventf(pod, annotations, v1.EventTypeWarning, Reason, evictMsg)
 	// this is a blocking call and should only return when the pod and its containers are killed.
 	klog.V(3).InfoS("Evicting pod", "pod", klog.KObj(pod), "podUID", pod.UID, "message", evictMsg)
+	_, otelSpan := m.tracer.Start(nil, "Eviction/EvictPod", trace.WithAttributes(
```

Comment:
including the eviction message as an attribute or as an event would be helpful

Reply:
Please plumb context to this function as well, instead of using nil.

Reply:
> including the eviction message as an attribute or as an event would be helpful

I decided to use event. Since in line 606 to 608 also use events.

Reply:
> Please plumb context to this function as well, instead of using nil.

Done